### PR TITLE
fix: fix overflow on be attacked

### PR DESCRIPTION
**Pull request by driftluo**
fix overflow on be attacked

### CODE REVIEW

File: secio/src/error.rs [hunk 1]
```diff
@@ -79,7 +79,7 @@ impl Into<io::Error> for SecioError {
     fn into(self) -> io::Error {
         match self {
             SecioError::IoError(e) => e,
-            e => io::Error::new(io::ErrorKind::BrokenPipe, error::Error::description(&e)),
+            e => io::Error::new(io::ErrorKind::BrokenPipe, e.to_string()),
```
Comment (quake) [inline secio/src/error.rs]: unrelated change for this bug fixing? suggest to sperate it to another PR.

Comment (driftluo) [reply secio/src/error.rs]: rustc 1.42 check warning

Comment (quake) [reply secio/src/error.rs]: > rustc 1.42 check warning

this repo's rust-toolchain file is still 1.41, I think that we can upgrade it with this fix in other PR.

Comment (driftluo) [reply secio/src/error.rs]: ok, reset it now
